### PR TITLE
fix(Alert): throw error if `secondaryAction` is defined without `primaryAction`

### LINKED ISSUE #725: Alerts: Allow `SecondaryAction` only when `PrimaryAction` is defined
Right now we don't throw any error if the consumers passes `SecondaryAction` without `PrimaryAction` we want to restrict this behaviour. We can't do it with TS but we shall add it as a runtime check.

### PR DESCRIPTION
**Pull request by shivam1646**
resolves #725

### CODE REVIEW

File: packages/blade/src/components/Alert/__tests__/Alert.native.test.tsx [hunk 1]
```diff
@@ -94,4 +94,23 @@ describe('<Alert />', () => {
...
+      renderWithTheme(
+        <Alert
+          description="Currently you can only accept payments in international currencies using PayPal."
+          // @ts-expect-error testing failure case when there is no primary action passed
+          actions={{
+            secondary: { text: 'Link', onClick: onClickSecondary },
+          }}
+        />,
+      );
+    } catch (error: unknown) {
```
Comment (divyanshu013) [inline packages/blade/src/components/Alert/__tests__/Alert.native.test.tsx:110]: Can't we simplify this with `expect(() => ...).toThrow()`?

File: packages/blade/src/components/Alert/__tests__/Alert.web.test.tsx [hunk 2]
```diff
@@ -103,4 +103,23 @@ describe('<Alert />', () => {
...
+      renderWithTheme(
+        <Alert
+          description="Currently you can only accept payments in international currencies using PayPal."
+          // @ts-expect-error testing failure case when there is no primary action passed
+          actions={{
+            secondary: { text: 'Link', onClick: onClickSecondary },
+          }}
+        />,
+      );
+    } catch (error: unknown) {
```
Comment (divyanshu013) [inline packages/blade/src/components/Alert/__tests__/Alert.web.test.tsx:119]: Here too

**Review by divyanshu013 — COMMENTED** [review]
A couple of comments on the tests

**Comment by shivam1646** [comment]
@divyanshu013 I have simplified the test cases as suggested.

**Comment by divyanshu013** [comment]
Holding off on merging this, pending some discussion which might make this change not required.

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

**Comment by shivam1646** [comment]
Hi @divyanshu013 @kamleshchandnani 
silly question but am I eligible for Hacktoberfest swag 🥺

**Comment by divyanshu013** [comment]
[Yes](https://github.com/razorpay/.github/blob/master/HACKTOBERFEST.md), you should be

**Comment by shivam1646** [comment]
Great, thanks. Do you know how to claim. There seem to be no such details about it [here](https://github.com/razorpay/.github/blob/master/HACKTOBERFEST.md).

**Comment by divyanshu013** [comment]
Can you please drop a DM to [RazorpayEngg](https://twitter.com/razorpayengg) on Twitter. Someone should get back from the team.

**Comment by shivam1646** [comment]
Thanks. I've sent a DM.

**Pull request merged**